src/BlockTypes/ProductRating.php

```diff
@@ -109,10 +110,13 @@ protected function render( $attributes, $content, $block ) {
 		);
 
 		if ( $product ) {
+			$classes_and_styles = StyleAttributesUtils::get_text_align_class_and_style( $attributes );
+
 			return sprintf(
-				'<div class="wc-block-components-product-rating wc-block-grid__product-rating">
+				'<div class="wc-block-components-product-rating wc-block-grid__product-rating %s">
 					%s
 				</div>',
+				$classes_and_styles['class'],
```

Comment:
When adding the `Product rating` block this `$classes_and_styles['class']` is `null`, so this line fails when trying to access `'class'` on `null` with this error:
```
Warning: Trying to access array offset on value of type null in woocommerce-blocks/src/BlockTypes/ProductRating.php on line 119